### DISCUSSION TITLE
There is no locale called en-EG but what i want is to have english running but the currency be EGP instead of ج.م.

### DISCUSSION THREAD

**Discussion by pshoukry** [elixir-cldr/cldr#226] [Q&A] [answered]
I basically need an english interface with local currency and time zone. Is that possible by setting cldr config ?

**Comment by kipcole9** [comment, marked as answer]
Peter, a language tag can convey quite a lot of information that isn't obvious at first. It is documented in [the BCP47 'U' extension](https://www.unicode.org/reports/tr35/#u_Extension).  Here's an example of using the `en` locale but with the Egyptian pound as the default currency and overriding the region to be `EG`.

```elixir
# English but preferring Egyptian Pound, Egypt as the region and Egypt/Cairo as the timezone
iex> {:ok, locale} = Cldr.validate_locale("en-u-cu-egp-rg-egzzzz-tz-egcai")
{:ok, #Cldr.LanguageTag<en-u-cu-egp-rg-egzzzz-tz-egcai [validated]>}

iex> Cldr.Currency.currency_from_locale(locale)
:EGP

iex> Cldr.Locale.territory_from_locale(locale)
:EG

iex> Cldr.Locale.timezone_from_locale(locale)
"Africa/Cairo"
```

That said, I'm not sure what you mean by `I basically need an english interface with local currency and time zone` so If you have any examples of what you're trying to achieve I can provide more detail.

**Reply by pshoukry** [reply]
Thank you that answers my question i will read more about it ... thank you. 

What i meant by my sentence is exctly that the language in en but i can choose currency or timezone based on the locale. the answer is spot on. Thank you sir!
